src/ipvs/ip_vs_proto_tcp.c

```diff
@@ -923,6 +924,8 @@ static int tcp_state_trans(struct dp_vs_proto *proto, struct dp_vs_conn *conn,
         }
     }
 
+    dp_vs_sync_conn_handler(conn, new_state);
+
```

Comment:
Since tcp trans will be called frequently，session sync packets will be so much. For example, DPVS can have performance for 100m pps, but for session sync, perhaps DPVS can only have less than 70m pps effectively，particularly for short connection. 
So I think session sync code should be include in "ifdef" and enabled in config.mk like pdump(https://github.com/mscbg/dpvs/commit/8087d1480b3ceb4f5b31fa34d5a3066bb97048d9). Could you please do this?

Reply:
Add CONFIG_DPVS_SYNC,  default value is disable